### ISSUE TITLE
Override positioning of Menu

### ISSUE THREAD

**Issue by ingesy** [jamesmfriedman/rmwc#83]
According to https://material.io/components/web/catalog/menus/ you should be able to override the automatic positioning of menus with css-classes.

Is this missing or is it some setting I'm not seeing?

I am on v0.0.1-rc13

**Comment by j-o-d-o** [comment]
Do you mean using css classes to chose the opening direction? Cause that is working as the classes are just passed to mdc-simple-menu. 

`<Menu className="mdc-simple-menu--open-from-bottom-right">...</Menu>`

**Comment by jamesmfriedman** [comment]
Hi @ingesy, we'll go ahead and add the props in to allow you to do that. In the meantime, as @j-o-d-o said, you can just pass the classNames to the menu.

```
// For now
<Menu className="mdc-simple-menu--open-from-bottom-right" />
```

```
// Next release will be...
<Menu openFromBottomRight />
```

**Comment by ingesy** [comment]
Nice, that will work nicely.

I tried to do what j-o-d-o suggested, but my real problem was that the Menu was wrapped with an MenuAnchor, which means automatic positioning.

**Comment by j-o-d-o** [comment]
@ingesy Now I see what you mean, sorry. I think this feature was just implemented in v0.28.0 (at least it is in the changelog for 0.28.0). https://github.com/material-components/material-components-web/commit/da56619

I am on it to check out what is needed for rmwc to update to mdc 0.28.0. The list is quite long thou might take a bit :/

**Comment by jamesmfriedman** [comment]
@ingesy this should be updated in V1. Check out the docs, you're going to want to play with the `anchorPosition` prop.
